packages/apidom-ls/src/config/openapi/parameter/lint/is-defined-within-path-template.ts

```diff
@@ -0,0 +1,23 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
+
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+import { OpenAPI2, OpenAPI3 } from '../../target-specs';
+
+const isDefinedWithinPathTemplate: LinterMeta = {
+  code: ApilintCodes.OPENAPI2_PARAMETER_FIELD_IS_DEFINED_WITHIN_PATH_TEMPLATE,
+  source: 'apilint',
+  message: 'parameter is not defined within path template',
+  severity: DiagnosticSeverity.Error,
+  linterFunction: 'apilintOpenAPIParameterFieldIsDefinedWithinPathTemplate',
+  marker: 'value',
+  targetSpecs: [...OpenAPI2, ...OpenAPI3],
+  conditions: [
```

Comment:
No need for this condition; This condition checks for valid path template, but Parameter Object has no direct connection to the path templating. (connection is via Path Item Object).


It is handled directly in the linting function: https://github.com/swagger-api/apidom/pull/3571/files#r1441457612